packages/mdc-textfield/_mixins.scss

```diff
@@ -486,24 +428,30 @@
   overflow: hidden;
 
   .mdc-text-field__input {
-    height: auto;
+    align-self: auto;
+    box-sizing: content-box;
+    height: 30px;
     margin: 0;
     padding: $padding-inset;
     padding-top: $padding-inset * 2;
     border: 1px solid transparent;
   }
 
   .mdc-floating-label {
+    @include mdc-rtl-reflexive-position(left, 1px);
     @include mdc-floating-label-fill-color($mdc-textarea-background);
-    @include mdc-rtl-reflexive(left, 1px, right, 0);
-    @include mdc-rtl-reflexive-property(margin, 8px, 0);
+    @include mdc-floating-label-width(auto);
 
     top: $label-offset-y;
     bottom: auto;
     margin-top: 2px;
-    padding: 12px 8px 8px 8px;
+    margin-left: 8px;
+    padding: 8px;
     line-height: 1.15;
-    pointer-events: none;
+    text-overflow: clip;
```

Comment:
I don't understand why these 4 styles were added - especially pointer-events: auto, didn't we recently intentionally set that to none to avoid the label stealing clicks from the input?

Reply:
I am trying to recreate the same effects that the text fields had before and so since more CSS was added in to the default .mdc-text-field I added styles to counteract the changes. However, Pointer-events actually does not counteract anything so i took that one out.